Source/1. Functional Programmingming/js/1_PartialApplication.js

```diff
@@ -0,0 +1,19 @@
+﻿var partialFn = (function () {
+    "use strict";
```

Comment:
imho using self calling functions here adds one more level of abstraction into a logic.